tinygrad/function.py

```diff
@@ -1,12 +1,24 @@
 """This is where the forwards and backwards passes live."""
 import math
 from typing import Tuple, Optional
-from tinygrad.helpers import argsort
+from tinygrad.helpers import argsort, getenv
 from tinygrad.dtype import dtypes, DType, sum_acc_dtype
 from tinygrad.ops import UnaryOps, BinaryOps, TernaryOps, ReduceOps
 from tinygrad.tensor import Function
 from tinygrad.lazy import LazyBuffer
 from tinygrad.shape.symbolic import sint
+from tinygrad.transcendental import xsin, xlog2, xexp2, is_dtype_transcendental_supported
+
+transcendental_supported_devices = ["CLANG", "LLVM"]
+def use_transcendental(d:LazyBuffer) -> bool:
+  # TRANSCENDENTAL=0 to always ignore.
+  # TRANSCENDENTAL=1 to run only in CLANG/LLVM (default).
+  # TRANSCENDENTAL=2 to always run it.
+  lv = getenv("TRANSCENDENTAL", 1)
```

Comment:
Should be a ContextVar in helpers so you can do "with(TRANSCENDENTAL=2):" 

Useful in the single kernel tests.

Reply:
@hikettei I see you added a ContextVar, but use it here as well `from tinygrad.helpers import TRANSCENDENTAL`